bundle-workflow/tests/tests_assemble_workflow/test_bundle_recorder.py

```diff
@@ -26,6 +26,14 @@ def setUp(self):
             manifest.build, "output_dir", "artifacts_dir"
         )
 
```

Comment:
This is only needed for one test. Add a `TestBundleRecorderDashboards` class (or similar name) in the same file with its own setup.